docs/symbols.md

```diff
@@ -4,38 +4,39 @@ List of symbols/abbreviations.
 
 | | |
 |-|-|
-| 3T-SBOM | Tool-to-Tool Software Bill of Material |
+| 3T-SBOM | Tool-to-Tool Software Bill of Materials Exchange |
 | ABNF | Augmented Backus–Naur form |
-| AI | Artificial Intelligence |
+| AI | Artificial intelligence |
 | BNF | Backus–Naur form |
-| BOM | Bill of Material |
-| CISA | Cybersecurity and Information Security Agency |
-| CISQ | Center for Information and Security Quality |
-| CPE | Common Package Enumeration |
-| CVE | Common Vulnerabilies and Exposures |
+| BOM | Bill of materials |
+| CISA | Cybersecurity and Infrastructure Security Agency |
+| CISQ | Consortium for Information & Software Quality |
+| CPE | Common Platform Enumeration |
+| CVE | Common Vulnerabilities and Exposures |
 | CVSS | Common Vulnerability Scoring System |
 | EPSS | Exploit Prediction Scoring System |
 | ISO | International Organization for Standardization |
-| JSON-LD | JavaScript Object Notation for Linked Data |
+| JSON-LD | JavaScript Object Notation for Linking Data |
 | KEV | Known Exploited Vulnerabilities |
-| ML | Machine Learning |
-| NISTIR | National Institute of Standards and Technology Internal/Interagency Reports |
+| ML | Machine learning |
```

Comment:
Put back the capital please.

Reply:
Kate, I have put back the capitalization so we can move on.


Although I feel that these are unnecessary capitalization.
All these AI, BOM, ML are generic names and not proper names. 
Manual of styles whether they are academic ones (like MPA, APA, or Chicago) or technical ones (see Microsoft and Google links below) suggest the use of all lowercase if the words are not proper names or product/service names (i.e. they can be capitalized only for cases that they are part of the product name).

https://developers.google.com/style/capitalization
https://learn.microsoft.com/en-us/style-guide/capitalization
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/artificial-intelligence-ai
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/m/machine-learning
https://learn.microsoft.com/en-us/dynamics365/supply-chain/production-control/bill-of-material-bom